src/components/learner-credit-management/search/CatalogSearch.jsx

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
 import algoliasearch from 'algoliasearch/lite';
 import { Configure, InstantSearch } from 'react-instantsearch-dom';
-import PropTypes from 'prop-types';
 
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { SearchHeader } from '@edx/frontend-enterprise-catalog-search';
 
 import { configuration } from '../../../config';
 import CatalogSearchResults from './CatalogSearchResults';
-import { SEARCH_RESULT_PAGE_SIZE } from '../data';
+import { SEARCH_RESULT_PAGE_SIZE, useBudgetId, useSubsidyAccessPolicy } from '../data';
 
-const CatalogSearch = ({ catalogUuid }) => {
+const CatalogSearch = () => {
   const searchClient = algoliasearch(configuration.ALGOLIA.APP_ID, configuration.ALGOLIA.SEARCH_API_KEY);
-  const searchFilters = `enterprise_catalog_uuids:${catalogUuid} AND learning_type:'course'`;
+  const { subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+  const searchFilters = `enterprise_catalog_uuids:${subsidyAccessPolicy.catalogUuid} AND learning_type:'course'`;
 
   return (
     <section>
       <FormattedMessage
         id="catalogs.enterpriseCatalogs.header"
-        defaultMessage="Budget associated catalog"
+        defaultMessage={subsidyAccessPolicy?.displayName || 'Overview'}
```

Comment:
[sanity check] Is this supposed to only be the `displayName` or is the `displayName` supposed to be concatenated with "{displayName} catalog" or "Overview catalog"?:

<img width="785" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/de030886-a98b-421c-8325-98cf02c93d3e">

Reply:
Good question! I clarified this with Jenn and we should be concatenating "`displayName` catalog" or just "Overview".